pkg/docker/config/config.go

```diff
@@ -85,27 +90,36 @@ func GetAuthentication(sys *types.SystemContext, registry string) (string, strin
 	}
 
 	dockerLegacyPath := filepath.Join(homedir.Get(), dockerLegacyHomePath)
```

Comment:
`dockerLegacyPath` can now be inlined into its only user.

Reply:
part of next push